src/AWS.Deploy.Common/Recipes/Validation/RangeValidator.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.\r
+// SPDX-License-Identifier: Apache-2.0
+
+namespace AWS.Deploy.Common.Recipes.Validation
+{
+    public class RangeValidator : IOptionSettingItemValidator
```

Comment:
can you add some docs to this class to explain what the validator is for?

Reply:
Fixed